### PR TITLE
Implement ability to soft-delete projects #4867

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/4867

Live Demo:
https://deploy-preview-248--atd-moped-main.netlify.app/moped/session/signin

Deletion:

Edit Title:

### CODE REVIEW

**Comment by sergiogcx** [comment]
@amenity @mateoclarke @mddilley Forgot to remove the WIP tag, just did.

**Review by mateoclarke — APPROVED** [review]
Good stuff Sergio! 

One small suggestion would be to make the dropdown options appear underneath the three dots icon instead of on top. Perhaps thats the default Material behavior, but here's an example of what I'm talking about from Github UI...

Screen Shot 2021-03-19 at 3 59 56 PM

If thats too much work or outside of Material defaults you can totally ignore me. 

Now we can...
delete

File: moped-editor/src/components/GridTable/GridTable.js [hunk 1]
```diff
@@ -144,15 +144,10 @@ const GridTable = ({ title, query }) => {
   // Set limit, offset and clear any 'Where' filters
   query.limit = pagination.limit;
   query.offset = pagination.offset;
-  query.cleanWhere();
+  query.cleanWhere(); // We need the latest version of this line btw
```
[the remark this answers is not in the document]
Comment (sergiogcx) [reply moped-editor/src/components/GridTable/GridTable.js:147]: @mateoclarke Ah, sorry yes I meant to say that it needs the latest version of Kickstand. I must have written this on here and forgot to actually be more specific.

File: moped-editor/src/views/projects/projectView/ProjectNameEditable.js [hunk 2]
```diff
@@ -42,10 +42,16 @@ const ProjectNameEditable = props => {
...
   const [projectName, setProjectName] = useState(initialProjectName);
-  const [projectNameBeforeEdit, setProjectNameBeforeEdit] = useState(projectName);
-  const [isEditing, setIsEditing] = useState(false);
+  const [projectNameBeforeEdit, setProjectNameBeforeEdit] = useState(
+    projectName
+  );
+  const [isEditing, setIsEditing] = useState(props?.isEditing ?? false);
   const [snackbarState, setSnackbarState] = useState(DEFAULT_SNACKBAR_STATE);
 
+  if (props?.isEditing === true && isEditing === false) {
```
[the remark this answers is not in the document]
Comment (sergiogcx) [reply moped-editor/src/views/projects/projectView/ProjectNameEditable.js:51]: Good call, I can make that change. 🙏

**Review by amenity — APPROVED** [review]
Soooo nice, @sergiogcx! So excited to start deleting 😍 🗑️  🔥

**Comment by amenity** [comment]
FWIW, sticking this in the backlog, @sergiogcx : https://github.com/cityofaustin/atd-data-tech/issues/5568

**Pull request merged**